### PR TITLE
Simple offline  signer

### PR DESCRIPTION
**Pull request by jacogr**

### CODE REVIEW

File: packages/signer-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# @polkadot/signer-cli
+
+A simple cli interface foir offline siging
```
Comment (joepetrowski) [inline packages/signer-cli/README.md]:
```suggestion
A simple cli interface for offline signing.
```

File: packages/signer-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+`yarn run:signer submit --account 5HNHXTw65dTNVGRdYkxFUpKcvmUYQMZHcDHmSKpuC8pvVEaN --ws wss://poc3-rpc.polkadot.io/ balances.transfer 5DkQbYAExs3M2sZgT1Ec3mKfZnAQCL4Dt9beTCknkCUn5jzo 123 `
+
+On execution it will prompt us with -
+
+```
+Payload: 0x040300ff4a83f1...a8239139ff3ff7c3f6
+Signature>
+```
+
+The `Payload` is the hex that needs to be signed. Pastign  the hex signature (followed by `Enter`) submit it to the chain.
```
Comment (joepetrowski) [inline packages/signer-cli/README.md]:
```suggestion
The `Payload` is the hex that needs to be signed. Paste the hex signature (followed by `Enter`) to submit it to the chain.
```

File: packages/signer-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+`yarn run:signer sign --account 5HNHXTw65dTNVGRdYkxFUpKcvmUYQMZHcDHmSKpuC8pvVEaN --seed "leaf .. rude" --type sr25519 0x040300ff4a83f1...a8239139ff3ff7c3f6`
+
+On executing, it will respond with -
+
+```
+Signature: 0xe6facf194a8e...413ce3155c2d1240b
+```
+
+Paste this signature into the  submission in the first terminal, and off we go.
```
Comment (joepetrowski) [inline packages/signer-cli/README.md]:
```suggestion
Paste this signature into the submission in the first terminal, and off we go.
```

File: packages/signer-cli/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# @polkadot/signer-cli
+
+A simple cli interface foir offline siging
+
+## Create transactions to sign
+
+In a terminal run the `submit` command with the following form -
```
Comment (joepetrowski) [inline packages/signer-cli/README.md]:
```suggestion
In a terminal, run the `submit` command with the following form -
```

File: packages/signer-cli/README.md (hunk 3 above)
Comment (jacogr) [reply packages/signer-cli/README.md]: My keyboard is dying (Mac), so I either do multiple spaces of stuff like '  . ', i.e. the "double spacing" turns into a ".". I've given up.

File: packages/signer-cli/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Signature>
+```
+
+The `Payload` is the hex that needs to be signed. Pasting the hex signature (followed by `Enter`) submits it to the chain.
+
+## Sign transactions
+
+In a terminal, run the `sign` command with the following form -
+
+`yarn run:signer sign --account <ss58> --seed <suri> --type <ed25519|sr25519> <hex signature>`
```
Comment (joepetrowski) [inline packages/signer-cli/README.md]: I think you mean the _payload,_ no?
```suggestion
`yarn run:signer sign --account <ss58> --seed <suri> --type <ed25519|sr25519> <hex payload>`
```

**Pull request merged**